coinmarketcap/test/adapter.test.ts

```diff
@@ -1,9 +1,13 @@
-const { assert } = require('chai')
-const { assertSuccess, assertError } = require('@chainlink/adapter-test-helpers')
-const { execute } = require('../adapter')
+import { assert } from 'chai'
+import { Requester } from '@chainlink/external-adapter'
+import { assertSuccess, assertError } from '@chainlink/adapter-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../src/adapter'
 
 describe('execute', () => {
   const jobID = '1'
+  const execute = makeExecute()
+  process.env.API_KEY = 'test_api_key'
```

Comment:
This is going to throw because: `makeExecute` -> `makeConfig` -> `Requester.getDefaultConfig` -> `util.getRandomRequiredEnv('API_KEY')`